lib/edit-site-page.php

```diff
@@ -145,17 +145,8 @@ function gutenberg_edit_site_init( $hook ) {
 	}
 	$settings['styles'] = gutenberg_get_editor_styles();
 
-	$settings['editSiteInitialState'] = array();
-
-	$settings['editSiteInitialState']['templateType'] = 'wp_template';
-	$settings['editSiteInitialState']['showOnFront']  = get_option( 'show_on_front' );
-	$settings['editSiteInitialState']['page']         = array(
-		'path'    => '/',
-		'context' => 'page' === $settings['editSiteInitialState']['showOnFront'] ? array(
-			'postType' => 'page',
-			'postId'   => get_option( 'page_on_front' ),
-		) : array(),
-	);
+	$settings['showOnFront'] = get_option( 'show_on_front' );
```

Comment:
I could not determine a way to fetch these options from the client without adding extra overrides. It would be helpful to have a way to fetch arbitrary options from the client. I'm not sure why these are not included in the default settings.

Reply:
I think we can just register these as settings now.